doc/en/components/grids/data-grid.md

```diff
@@ -1043,7 +976,7 @@ function fetchData(url: string): void {
       .then(data => onDataLoaded(data));
 }
 function onDataLoaded(jsonData: any[]) {
-    gridRef.current.data = jsonData;
+    grid.current.data = jsonData;
```

Comment:
Should we show this with state instead of using the ref. Probably, the better react-ish way